apps/graphql/src/definitions/Project.ts

```diff
@@ -33,6 +34,8 @@ export const typeDefs = gql`
     builds(first: Int!, after: Int!): BuildConnection!
     "A single build linked to the repository"
     build(number: Int!): Build
+    "Project has Tests"
+    hasTests: Boolean!
```

Comment:
I removed it because finally it is not the philosophy of GraphQL. If request is perf the totalCount should be perf too.